docker-compose.yml

```diff
@@ -14,3 +14,12 @@ services:
     build:
       context: .
       dockerfile: mat-process-api.Tests/Dockerfile
+    environment:
+        - MONGO_CONN_STRING=mongodb://mongo-db:27017/?gssapiServiceName=mongodb
```

Comment:
According to MongoDB documentation, the gssapiServiceName is 
> `Only required if the service does not use the default name of mongodb.`

We might not need this part of the connection string options, since what it's doing, is just specifying the default name of the service anyway.

https://docs.mongodb.com/manual/reference/program/mongo/#cmdoption-mongo-gssapiservicename